src/test/regress/bin/diff

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Our custom diff tool which normalizes result and expected files before
+# doing the actual diff. Rules for normalization are in normalize.sed. See
+# the comments there for more information.
+#
+# We only normalize tests listed in normalized_tests.lst. We use the bare
+# diff for other tests.
+#
+# Note that src/test/regress/Makefile adds this directory to $PATH so
+# pg_regress uses this diff tool instead of the system diff tool.
+
+file1="${@:(-2):1}"
+file2="${@:(-1):1}"
+test=$(basename $file1 .out)
+args=${@:1:$#-2}
+BASEDIR=$(dirname "$0")
+
+# whereis searches for standard unix places before $PATH. So select the first
+# entry as the original diff tool.
+DIFF=$(whereis diff | sed "s/diff://g" | awk '{print $1}')
```

Comment:
Minor: Should we fallback to `/usr/bin/diff` when ` if [ -z "$DIFF" ];`? The error would at least be more reasonable.

Reply:
done